src/Germany/Mews.Fiscalizations.Germany/V2/FiskalyClient.cs

```diff
@@ -35,7 +35,7 @@ public Task<ResponseResult<Model.Client>> GetClientAsync(AccessToken token, Guid
         public Task<ResponseResult<IEnumerable<Model.Client>>> GetAllTssClientsAsync(AccessToken token, Guid tssId)
         {
             return Client.GetResponseAsync<Dto.MultipleClientResponse, IEnumerable<Model.Client>>(
-                endpoint: $"tss/{tssId}/client",
+                endpoint: $"tss/{tssId}/client?state=REGISTERED",
```

Comment:
Same here, don't like hardcoding this into the endpoint. So either enum or rename method.